modules/dxp/apps/commerce-avalara/commerce-avalara-lang/src/main/resources/content/Language.properties

```
@@ -1,9 +1,21 @@
 account-number=Account Number
 avalara=Avalara
 avalara-tax-rate-description=Avalara Type Rate Description
-commerce-avalara-connecto-configuration-name=Commerce Avalara Connector Configuration Name
+avalara-tax-rates=Avalara Tax Rates
+avalara.tax.codes.commerce.health.status.description=Check if there are Avalara Tax Codes available to map to Tax Categories
```

Should end with a period if its a sentence.
```suggestion
avalara.tax.codes.commerce.health.status.description=Check if there are Avalara tax codes available to map to tax categories.
```